src/containers/shared/components/Transaction/DelegateSet/Description.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionDescriptionProps } from '../types'
+import { DelegateSet } from './types'
+import { Account } from '../../Account'
+
+export const Description = ({
+  data,
+}: TransactionDescriptionProps<DelegateSet>) => {
+  const { t } = useTranslation()
+  const { tx } = data
+  const {
+    Account: account,
+    Authorize: authorize,
+    Permissions: permissions,
+  } = tx
+  return (
+    <>
+      <Account account={account} />
+      {` `}
+      <span>{t('delegate')}</span>
+      {` `}
```

Comment:
ditto: can't you can remove the ``{` `}`` and just do a normal whitespace in between the elements?

Reply:
All of these can be fixed by what Mayukha suggested below. I will be on it